c/go/Makefile

```diff
@@ -1,16 +1,17 @@
 all: build
 
-build: libvirtblocks_c_go.la capi.lo
+build: libvirtblocks_c_go.la
 
 gobuild:
 	go build -buildmode c-archive -o libvirtblocks_c_go.a capi.go
 
-capi.lo: capi.c gobuild
-	libtool --mode=compile --tag=CC gcc -c capi.c
+capi.o: capi.c gobuild
+	gcc -c capi.c -o capi.o
```

Comment:
Can we really just ditch `libtool` like this? I'm a bit skeptical:

```
$ libtool --mode=compile --tag=CC gcc -c -o capi.lo capi.c
libtool: compile:  gcc -c capi.c  -fPIC -DPIC -o .libs/capi.o
libtool: compile:  gcc -c capi.c -o capi.o >/dev/null 2>&1
```

We've just lost `-fPIC -DPIC` which AFAIK we really want, since we're ultimately going to incorporate the object file into a shared library.

Reply:
Good point, I forgot about those.